lib/web-portal.ts

```diff
@@ -129,6 +146,35 @@ export class WebPortalStack extends cdk.NestedStack {
             ])],
         })
 
+        // lambda function to generate iso download links
+        fs.copyFileSync(path.join(__dirname, '../web-portal/mirror-web/geninfo/genisolist.ini'), path.join(__dirname, '../web-portal/genisolist/genisolist.ini'));
+        fs.copyFileSync(path.join(__dirname, '../web-portal/mirror-web/geninfo/genisolist.py'), path.join(__dirname, '../web-portal/genisolist/genisolist.py'));
+        const fileSystem = efs.FileSystem.fromFileSystemAttributes(this, `FileSystem`, {
+            fileSystemId: props.fileSystemId,
+            securityGroup: ec2.SecurityGroup.fromSecurityGroupId(this, `FileSystemSG`, props.fileSystemSGId)
+        });
+        const accessPoint = new efs.AccessPoint(this, `${usage}GenIsoLambdaAccessPoint`, {
```

Comment:
How about moving the creation of access point to storage stack?

Reply:
There is no way to construct a `efs.AccessPoint` solely from its id.

From `https://github.com/aws/aws-cdk/blob/bbf23a86af1aa527999f7df2a92c54bbdf7fa8fe/packages/%40aws-cdk/aws-lambda/lib/filesystem.ts#L53`, it needs `efs.AccessPoint` type, but the lookup function `https://github.com/aws/aws-cdk/blob/bbf23a86af1aa527999f7df2a92c54bbdf7fa8fe/packages/%40aws-cdk/aws-efs/lib/access-point.ts#L118` returns `efs.IAccessPoint`.